cloudinit/ssh_util.py

```diff
@@ -44,7 +44,7 @@
 DISABLE_USER_OPTS = (
     "no-port-forwarding,no-agent-forwarding,"
     "no-X11-forwarding,command=\"echo \'Please login as the user \\\"$USER\\\""
-    " rather than the user \\\"$DISABLE_USER\\\".\';echo;sleep 10\"")
+    " rather than the user \\\"$DISABLE_USER\\\".\';echo;sleep 10;exit 1\"")
```

Comment:
I wouldn't pick 1.
To hard to grep for or search for.

exiting 101 would be easier to grep for.  Pick your favorite unlikely value here.  I just don't like 1 as I exit 1 is so common for anything.

Reply:
perhaps we could return the same value as ssh itself would?

Reply:
As a reference:
```
# ~/develop/cloud-init $ grep 101 * -R|wc -l
83                             
# ~/develop/cloud-init $ grep 255 * -R|wc -l
334
```
But:
```
# ~/develop/cloud-init $ grep 142 * -R|wc -l
5
```
So, I'm gonna randomly choose `142` just because it's easier to grep. Yes, I tried lots of numbers but couldn't find a unique one.